x-pack/plugins/screenshotting/server/layouts/create_layout.ts

```diff
@@ -40,7 +40,7 @@ export function createLayout({ id, dimensions, selectors, ...config }: LayoutPar
 
   if (dimensions) {
     if (layoutId === 'preserve_layout') {
-      return new PreserveLayout(sanitizeLayout(dimensions), selectors);
+      return new PreserveLayout(sanitizeLayout(dimensions), selectors, config.zoom);
```

Comment:
nit: can we rename the `config` variable to something like `layout` or `layoutOpts`?